### PR TITLE
feat(ui): use search bar to kick off project search

### PR DESCRIPTION
**Pull request by sarahscott**
🚧  depends on work in fintolex's branch 🚧

### CODE REVIEW

**Comment by FintanH** [comment]
So as I mentioned `Instant` has no `Serialize` impl, however this lib provides one https://docs.rs/serde_millis/0.1.1/serde_millis/

Maybe what we _could_ do is try:
```rust
impl Serialize for WaitingRoom<Instant, Duration>
```

writing it manually and for the timestamp field we just call https://docs.rs/serde_millis/0.1.1/serde_millis/fn.serialize.html. I believe that should work. So you can disregard my comment about making a new `WaitingRoom` newtype around `Instant` + `Duration`.

File: ui/src/search.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import { writable } from "svelte/store";
 
+import * as api from "./api";
 import * as event from "./event";
-
+import * as remote from "./remote";
+// import * as waitingRoom from "./waitingRoom";
```
Comment (rudolfs) [inline ui/src/search.ts:6]:
```suggestion
```

Comment (FintanH) [reply ui/src/search.ts:6]: I wanna leave it there so we reintroduce it when we _actually_ respond with waiting room data, if that's cool with you?

File: ui/Screen/Profile/Tracking.svelte [hunk 2]
```diff
@@ -4,55 +4,39 @@
...
-        description: "Lemuel is a villager from Butcher Creek.",
-        maintainers: [],
-      },
-      registration: false,
-      shareableEntityIdentifier: "lemuel@38590438594",
-      maintainers: [],
-    },
-  ];
+  fetchTracked();
+  $: console.log($tracked);
```
Comment (rudolfs) [inline ui/Screen/Profile/Tracking.svelte:16]: What about this one?

**Review by xla — APPROVED** [review]
🏖 🎤 🔋 🍌

**Review by FintanH — DISMISSED** [review]

**Review by xla — APPROVED** [review]

**Review by FintanH — APPROVED** [review]

File: proxy/coco/src/request.rs [hunk 3]
```diff
@@ -71,6 +71,53 @@ pub struct Request<S, T> {
...
+    T: Serialize,
+{
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        let mut state = serializer.serialize_struct("Request", 4)?;
+        state.serialize_field("urn", &self.urn)?;
+        state.serialize_field("attempts", &self.attempts)?;
+        state.serialize_field("timestamp", &WithMillis(self.timestamp))?;
```
Comment (FintanH) [inline proxy/coco/src/request.rs:98]: @sarahscott I think we could just do `serde_millis::serialize(self.timestamp)` here and remove  `WithMillis`

**Pull request merged**